common/lib/container-definitions/src/loader.ts

```diff
@@ -128,28 +128,45 @@ export interface IContainerEvents extends IEvent {
 
 /**
  * Namespace for the different connection states a container can be in
+ * PLEASE NOTE: The sequence of the numerical values does no correspond to the typical connection state progression
  */
 export namespace ConnectionState {
     /**
-     * The document is no longer connected to the delta server
+     * The container is not connected to the delta server
+     * Note - When in this state the container may be about to reconnect,
+     * or may remain disconnected until explicitly told to connect.
      */
     export type Disconnected = 0;
 
     /**
-     * The document has an inbound connection but is still pending for outbound deltas
+     * The container is disconnected but actively trying to establish a new connection
+     * PLEASE NOTE that this numerical value falls out of the order you may expect for this state
      */
+     export type EstablishingConnection = 3;
+
+     /**
+     * The container has an inbound connection only, and is catching up to the latest known state from the service.
+     */
+    export type CatchingUp = 1;
+
+    /** @deprecated - Use ConnectionState.CatchingUp */
```

Comment:
It might be helpful to include a little context here. We can emphasize that `Connecting` and `CatchingUp` represent the same state and this is a rename. We can also note that `Connecting` will be removed in the future and give the version we expect it to be complete (I believe it should be v2.0?)